python/sglang/srt/model_executor/model_runner.py

```diff
@@ -418,13 +441,14 @@ def init_memory_pool(
             self.token_to_kv_pool = MHATokenToKVPool(
                 self.max_total_num_tokens,
                 dtype=self.kv_cache_dtype,
+                device=self.device_config.device_type,
                 head_num=self.model_config.get_num_kv_heads(self.tp_size),
                 head_dim=self.model_config.head_dim,
                 layer_num=self.model_config.num_hidden_layers,
             )
         logger.info(
             f"Memory pool end. "
-            f"avail mem={get_available_gpu_memory(self.gpu_id):.2f} GB"
+            f"avail mem={get_available_gpu_memory(self.device_config.device_type, self.gpu_id):.2f} GB  on {self.device_config.device_type}:{self.gpu_id}"
```

Comment:
Add a `device_type = self.device_config.device_type` at this function's beginning to shorten this line.

Reply:
Done